app/src/main/java/org/jellyfin/androidtv/playback/overlay/CustomActionClickedHandler.java

```diff
@@ -0,0 +1,147 @@
+package org.jellyfin.androidtv.playback.overlay;
+
+import android.content.Context;
+import android.view.Gravity;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.PopupMenu;
+
+import org.jellyfin.androidtv.R;
+import org.jellyfin.androidtv.TvApp;
+import org.jellyfin.androidtv.livetv.TvManager;
+import org.jellyfin.androidtv.model.compat.SubtitleStreamInfo;
+import org.jellyfin.androidtv.playback.CustomPlaybackOverlayFragment;
+import org.jellyfin.androidtv.playback.PlaybackController;
+import org.jellyfin.androidtv.playback.VideoManager;
+import org.jellyfin.androidtv.ui.AudioDelayPopup;
+import org.jellyfin.androidtv.ui.ValueChangedListener;
+import org.jellyfin.androidtv.util.Utils;
+import org.jellyfin.apiclient.model.entities.MediaStream;
+
+import java.util.List;
+
+public class CustomActionClickedHandler {
```

Comment:
I don't really like how this one class is responsible for all click handlers, I'd rather see them added to their respective action class.